site/source/_posts/2020-01-13-announcing-embark-5.md

```diff
@@ -0,0 +1,214 @@
+title: Introducing Embark 5
+author: pascal_precht
+summary: "About half a year after our last stable release, we've now published Embark version 5 with lots of features, improvements and fixes. Read on for more information!"
+categories:
+  - announcements
+layout: blog-post
+---
+
+If you've been following the development of Embark you're probably aware that we regularly put out alpha and beta releases for upcoming major or feature versions of Embark. In the past ~6 months since the release of Embark 4, we've published 10 alpha releases and one beta release for Embark 5 and today we're happy to announce the Embark 5 stable release!
+
+In this post we'll be looking at some of the main changes and features to get up and running with v5. Notice that we've add a [migration guide](https://embark.status.im/docs/migrating_from_3.x.html#Updating-to-v5) to our official docs as well.
+
+## New Features
+
+Let's first start with new features that have been introduced in Embark 5.
+
+### Whisper client configuration
+
+Prior to Embark 5, Embark would use the same blockchain client to enable blockchain features and communication features for Whisper. With Embark 5, Embark spins up a separate client for Whisper. This also ensures that that Whisper service stays in tact when the blockchain service is turned off and vice versa.
+
+To specify a client, use the new `client` configuration property which defaults to `geth`:
+
+```js
+// communication.js
+
+...
+default: {
+  ...
+  client: "geth" // can be either 'geth' or 'parity'
+},
+...
+
+```
+
+### Support for Dynamic Addresses
+
+If you're using Embark already, you're probably aware that there are many different ways to [configure your Smart Contracts](/docs/contracts_configuration.html). One of the things that can be configured is the `address` of a Smart Contract. Usually the address will be determined after a Smart Contract has been deployed. In other cases, what we want to do is simply specifying the address because the Smart Contract is already deployed to the network.
```

Comment:
```suggestion
If you're using Embark already, you're probably aware that there are many different ways to [configure your Smart Contracts](/docs/contracts_configuration.html). One of the things that can be configured is the `address` of a Smart Contract. Usually the address will be determined after a Smart Contract has been deployed. In other cases, what we want to do is simply specify the address because the Smart Contract is already deployed to the network.
```